packages/react-components/src/components/CameraButton.tsx

```diff
@@ -49,13 +50,18 @@ export const CameraButton = (props: CameraButtonProps): JSX.Element => {
     return props?.checked ? <CallVideoIcon key={'videoIconKey'} /> : <CallVideoOffIcon key={'videoOffIconKey'} />;
   };
 
-  const defaultRenderText = (props?: IButtonProps): JSX.Element => {
-    return (
-      <Label key={'videoLabelKey'} className={mergeStyles(controlButtonLabelStyles, props?.styles?.label)}>
-        {props?.checked ? 'Turn off' : 'Turn on'}
-      </Label>
-    );
-  };
+  const strings = useLocale().strings;
+
+  const defaultRenderText = useMemo(
+    () => (props?: IButtonProps): JSX.Element => {
+      return (
+        <Label key={'videoLabelKey'} className={mergeStyles(controlButtonLabelStyles, props?.styles?.label)}>
+          {props?.checked ? strings.camera_button_on_text : strings.camera_button_off_text}
+        </Label>
+      );
+    },
+    [strings]
```

Comment:
Will this still rerender the component when the props have changed? (props.checked and props.styles.label)

Reply:
I don't think it does because I am memoizing a function that depends only on the strings values but does not depend on the values of props.checked and props.styles.label since props is an argument of this memoized function.